test/WebJobs.Script.Tests/Workers/Rpc/RpcWorkerConfigFactoryTests.cs

```diff
@@ -153,6 +153,7 @@ public void DefaultWorkerConfigs_Overrides_DefaultWorkerRuntimeVersion_AppSettin
             var config = configBuilder.Build();
             var scriptSettingsManager = new ScriptSettingsManager(config);
             var testLogger = new TestLogger("test");
+            _testEnvironment.SetEnvironmentVariable(EnvironmentSettingNames.AzureWebsitePlaceholderMode, "1");
```

Comment:
This test uses the worker.config from the referenced language worker packages. The .NET Isolated worker config has profile condition to check for this env variable. See https://github.com/Azure/azure-functions-dotnet-worker/pull/2552 for details.

Reply:
Does this impact non-dotnet tests? If so, can we scope this to be set only for dotnet isolated?

Reply:
This test is not worker runtime specific. It validates the outer portion of code which gets all the workers.